src/test/java/ItemTest.java

```diff
@@ -0,0 +1,46 @@
+import controllers.ItemController;
+import entities.Item;
+import repositories.ItemRepository;
+import utils.IoC;
+
+public class ItemTest {
+
+    public static void main(String[] args) {
+
+        IoC ioc = IoC.getInstance();
+
+        ItemController itemController = ioc.getInstance("ItemController");
```

Comment:
Use `var` instead if this is java 11. Don't forget the generics.

You could do:

```
var itemController = ioc.<ItemController>getInstance("ItemController")
```